flow/flow_test.go

```diff
@@ -273,9 +285,8 @@ func validateAllParentChains(t *testing.T, table *Table) {
 
 func flowsFromPCAP(t *testing.T, filename string, linkType layers.LinkType, bpf *BPF) []*Flow {
 	table := NewTable(nil, nil, NewEnhancerPipeline(), "", TableOpts{})
-
+	table.SetSynCapture(true)
```

Comment:
I tried, it didnt work. Somehow in the test it sees a different config variable... I did exactly what you type and printed the valuse right before calling table after and inside the code of table, and I see that they work on different config instances... So this was my hack to overcome the problem.
This function is not totally not needed, I think in the future we might want to add some UI button that could change SYN/FIN capturing flag, then it will be useful not only in the tests